prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1120,44 +1120,28 @@ static ssize_t rxr_ep_cancel(fid_t fid_ep, void *context)
  */
 static int efa_set_fi_hmem_p2p_opt(struct efa_ep *efa_ep, int opt)
 {
```

Comment:
I think this function need to document what it is doing, and what the possible return values are.

Reply:
Is the [existing comment](https://github.com/ofiwg/libfabric/blob/55e77cda043849f47dcb9780e947b4b28fd5d4de/prov/efa/src/rxr/rxr_ep.c#L1114-L1120) not sufficient?

```c
/*
 * Validate p2p opt passed by the user and set the endpoint option if it is
 * valid. If the option chosen is invalid or not supported, return an error.
 *
 * @param[in]	efa_ep	efa endpoint
 * @return 	0 on success, negative errno on error
 */
```

Reply:
Thank you for expanding the comment here to specify which error messages mean what.